tests/codeception/acceptance/SystemCest.php

```diff
@@ -14,9 +14,9 @@ public function testSiteStatus(AcceptanceTester $I) {
     $I->runDrush('xmlsitemap:rebuild');
     $I->logInWithRole('administrator');
     $I->amOnPage('/admin/reports/status');
-    $I->canSee('10.3', '.system-status-general-info');
+    $I->canSee('10.4', '.system-status-general-info');
     if ($I->grabMultiple('.system-status-counter--error')) {
-      $I->canSee('1 Error', '.system-status-counter--error');
+      $I->canSee('2 Error', '.system-status-counter--error');
```

Comment:
What's the additional error being accounted for here?

Reply:
@joegl Good question. I don't see on on my local or on the dev, but the tests see an additional one.

Reply:
@joegl The test wasn't passing without it, but now it is.